### PR TITLE
Doc and import improvement #3

### PR DESCRIPTION
**Pull request by jlaehne**
### Description of the change
Follow up on #49 & #59, adresses #38. For readers with folder names from N...P.

### Progress of the PR
- [X] Change implemented (can be split into several points),
- [X] update docstring (if appropriate),
- [X] update user guide (if appropriate),
- [ ] add an changelog entry in the `upcoming_changes` folder (see [`upcoming_changes/README.rst`](https://github.com/hyperspy/rosettasciio/blob/main/upcoming_changes/README.rst)),
- [ ] Check formatting changelog entry in the `readthedocs` doc build of this PR (link in github checks)
- [X] add tests,
- [X] ready for review.

### CODE REVIEW

File: rsciio/ripple/_api.py [hunk 1]
```diff
@@ -247,109 +253,23 @@ def read_raw(rpl_info, fp, mmap_mode="c"):
...
+    ripple (.rpl) file and reads the data from the corresponding raw (.raw) file;
+    or, read a raw file if the dictionary ``rpl_info`` is provided.
 
+    Parameters
+    ----------
+    %s
+    %s
+    enconding : str, optional
+        The default encoding is ``latin-1``. Different file encodings, such as
+        ``utf8`` can be set via this argument.
```
Comment (ericpre) [inline rsciio/ripple/_api.py:266]:
```suggestion
       The encoding used to read the content of the ``rpl`` file. The default
       encoding is ``latin-1``. Different file encodings, such as ``utf8`` can be
       set via this argument.
```

File: rsciio/ripple/_api.py [hunk 2]
```diff
@@ -536,7 +456,18 @@ def file_reader(
...
+
 def file_writer(filename, signal, encoding="latin-1", *args, **kwds):
+    """Writes a Lispix (https://www.nist.gov/services-resources/software/lispix)
+    ripple (.rpl) file and saves the data in a corresponding raw (.raw) file.
+
+    Parameters
+    ----------
+    %s
+    %s
+    """
```
Comment (ericpre) [inline rsciio/ripple/_api.py:470]: Add encoding arguments, something along the line of: 
```python
       The encoding used to read the content of the ``rpl`` file. The default
       encoding is ``latin-1``. Different file encodings, such as ``utf8`` can be
       set via this argument.
```

File: rsciio/ripple/_api.py [hunk 3]
```diff
@@ -247,109 +253,23 @@ def read_raw(rpl_info, fp, mmap_mode="c"):
...
-
-    Any number of spaces can go along with each tab.
+    """Parses a Lispix (https://www.nist.gov/services-resources/software/lispix)
+    ripple (.rpl) file and reads the data from the corresponding raw (.raw) file;
+    or, read a raw file if the dictionary ``rpl_info`` is provided.
 
+    Parameters
+    ----------
+    %s
+    %s
```
Comment (ericpre) [inline rsciio/ripple/_api.py:263]: Add `rpl_info` docstrings?

File: docs/supported_formats/ripple.rst [hunk 4]
```diff
@@ -19,24 +19,19 @@ It can also be used to exchange data with Bruker and used in
...
 
 
+API functions
+^^^^^^^^^^^^^
+
+.. automodule:: rsciio.ripple
+   :members:
+
+
 Format description
```
Comment (ericpre) [inline docs/supported_formats/ripple.rst:35]: Shall this section be moved to the `File Specifications` section of the documentation with a link in the introduction section.

File: docs/supported_formats/nexus.rst [hunk 5]
```diff
@@ -340,40 +304,9 @@ The output will be arranged by signal name:
...
-    original_metadata, signal data, axes, metadata and learning_results
+    Signals saved as ``.nxs`` by this plugin can be loaded normally in HyperSpy
+    and the original_metadata, signal data, axes, metadata and learning_results
     will be restored. Model information is not currently stored.
     NeXus does not store how the data should be displayed.
     To preserve the signal details an additional navigation attribute
     is added to each axis to indicate if it is a navigation axis.
-
-
-Inspecting
```
[the remark this answers is not in the document]
Comment (jlaehne) [reply docs/supported_formats/nexus.rst:351]: OK, I placed them under `utils.hdf5`.

File: docs/supported_formats/ripple.rst (hunk 4 above)
Comment (jlaehne) [reply docs/supported_formats/ripple.rst]: I created a dedicated page under `File specifications`.

**Pull request merged**